lib/create.js

```diff
@@ -122,26 +137,27 @@ export default async () => {
     );
 
     // setup npm dependencies for plugin
-    console.log(chalk.green('Setting up node_modules...'));
+    logger.info(chalk.green('Setting up node_modules...'));
     execSync('npm install', { stdio: [0, 1, 2], cwd: destinationPath });
     if (dependencies) {
-      console.log(chalk.green('Installing plugin dependencies from npm'));
+      logger.info(chalk.green('Installing plugin dependencies from npm'));
       execSync(`npm install --save ${dependencies.replace(/,/g, '')}`, {
```

Comment:
Would it be a good idea validate dependencies before just passing it into `npm install`?

Reply:
I think this is along the same lines as the other comment. I don't think there's any more need for us to validate dependencies since that's basically what `npm install` will do and this is really just a wrapper around a user running `npm install` themselves. I can't see a circumstance where we need to stop someone from injecting arbitrary code execution on themselves that they wouldn't couldn't otherwise do from including it in a normal `npm install` command. There is also some validation that already happens in `questions.js` where this is set. I can add an extra check using npm's `validate` package though to make sure the names don't have any typos.